geo-data-server/src/main/java/com/powsybl/geodata/server/GeoDataService.java

```diff
@@ -210,12 +216,20 @@ private static SubstationGeoData calculateCentroidGeoData(Substation substation,
         return neighbours;
     }
 
-    public void saveSubstations(List<SubstationGeoData> substationsGeoData) {
+    void saveSubstations(List<SubstationGeoData> substationsGeoData) {
+        String listSize = substationsGeoData.size() + "";
```

Comment:
Same here we only print the size, there is no potiential log forging issue?

Reply:
Sonar explanation :
User provided data, such as URL parameters, POST data payloads or cookies, should always be considered **untrusted and tainted**. Applications logging tainted data could enable an attacker to inject characters that would break the log file pattern. This could be used to block monitors and SIEM (Security Information and Event Management) systems from detecting other malicious events.

Here, substationsGeoData is considered as unstrusted parameter. sonar prevent use the result of substationsGeoData.size() is the log trace.

Reply:
X.replaceAll("\"[\\n|\\r|\\t]\"", "_") should be applied before using the result as a log input.

Reply:
But here we only log the size, so we cannot have line breaks added to logs?

Reply:
Alright, that doesn't make sense!  should we ignore that issue ?